src/metrics/models.py

```diff
@@ -55,6 +55,45 @@ def remove_one_download(self):
         self.save()
 
 
+def object_types():
+    return (
+        ('book', 'Book'),
+        ('article', 'Article'),
+    )
+
+
+class ForwardLink(models.Model):
+    article = models.ForeignKey('submission.Article', blank=True, null=True)
+    doi = models.CharField(max_length=255)
+    object_type = models.CharField(max_length=10, choices=object_types())
+    year = models.CharField(max_length=5)
```

Comment:
This model should be declared as an abstract base (Its common to name them accordingly as well: `AbstractForwardLink`)
https://docs.djangoproject.com/en/1.11/topics/db/models/#abstract-base-classes